__tests__/integration/nano-contracts.test.js

```diff
@@ -115,7 +114,7 @@ describe('nano contract routes', () => {
         method: 'bet',
         data: {
           args: [
-            bufferUtils.bufferToHex(address3Obj.decode()),
```

Comment:
chore: We should also remove the `address3Obj` declaration on line 108

Reply:
Fixed in https://github.com/HathorNetwork/hathor-wallet-headless/pull/481/commits/3a568fd387da4db800cfa6edb5c1357895bfad98